Compiler/NFFrontEnd/NFFunction.mo

```diff
@@ -202,7 +229,9 @@ uniontype Function
     end try;
 
     // Look up the function and create a cref for it.
-    (node, nodes, found_scope) := Lookup.lookupFunctionName(functionName, scope, info);
+    //(node, nodes, found_scope) := Lookup.lookupFunctionName(functionName, scope, info);
```

Comment:
This will break some of the test cases since assertCallable doesn't handle all the error cases. It would be better to just fix LookupState.assert to handle record constructors correctly instead, and keep all the error checking in one place.